test/commands/logic/run/test.nut.ts

```diff
@@ -0,0 +1,206 @@
+    const flowTestsDir = path.join(session.project.dir, 'force-app', 'main', 'default', 'flowtests');
+    const targetTestFile = path.join(flowTestsDir, 'test_opportunity_updates.flowtest-meta.xml');
+    fs.mkdirSync(flowTestsDir, { recursive: true });
+    fs.copyFileSync(flowTestXml, targetTestFile);  const sfdxProjectPath = path.join(session.project.dir, 'sfdx-project.json');
+  
+    // We need to update the sourceApiVersion to 65.0 because the changes ub the api are not supported in 64.0
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const sfdxProject = JSON.parse(fs.readFileSync(sfdxProjectPath, 'utf8'));
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-argument
+    sfdxProject.sourceApiVersion = parseInt(sfdxProject.sourceApiVersion, 10) < 65 ? '65.0' : sfdxProject.sourceApiVersion;
+    fs.writeFileSync(sfdxProjectPath, JSON.stringify(sfdxProject, null, 2));
+
+    execCmd('project:deploy:start -o org --source-dir force-app', { ensureExitCode: 0, cli: 'sf' });
+  });
+
+  after(async () => {
+    await session?.zip(undefined, 'artifacts');
+    await session?.clean();
+  });
+
+  describe('--test-category', () => {
+    it('will run tests with Apex category', async () => {
+      const result = execCmd('logic:run:test --test-category Apex --test-level RunLocalTests --wait 10', { 
+        ensureExitCode: 0 
+      }).shellOutput.stdout;
+      expect(result).to.include('CATEGORY');
+      expect(result).to.include('Apex');
+      expect(result).to.not.include('Flow');
+    });
+
+    it('will run tests with Flow category', async () => {
+      const result = execCmd('logic:run:test --test-category Flow --test-level RunLocalTests --wait 10', { 
+        ensureExitCode: 0 
+      }).shellOutput.stdout;
+      expect(result).to.include('CATEGORY');
+      expect(result).to.include('Flow');
+      expect(result).to.not.include('Apex');
+    });
+
+    it('will run tests with multiple categories', async () => {
+      const result = execCmd('logic:run:test --test-category Apex --test-category Flow --test-level RunLocalTests --wait 10', { 
+        ensureExitCode: 0 
+      }).shellOutput.stdout;
+      expect(result).to.include('CATEGORY');
+      expect(result).to.include('Apex');
+      expect(result).to.include('Flow');
+    });
+  });
+
+  describe('--class-names', () => {
+    it('will run specified class', async () => {
+      const result = execCmd('logic:run:test --class-names GeocodingServiceTest --wait 10', { 
+        ensureExitCode: 0 
+      }).shellOutput.stdout;
+      expect(result).to.match(/Tests Ran\s+3/);
+      expect(result).to.include('GeocodingServiceTest');
+      expect(result).to.include('CATEGORY');
+      expect(result).to.include('Apex');
```

Comment:
we should add an `expect(result).to.not.include('Flow');` this would apply to other tests where it's one or the other, that way we verify the behavior doesn't change without breaking tests